tests/test_image_analysis.py

```diff
@@ -72,3 +73,13 @@ def test_get_avg_psd():
 
     assert avg_psd_1d is not None
     assert type(std_psd_1d) == np.ndarray
+
+
+def test_get_wt_coeff_var():
+    cur_dir = pathlib.Path(__file__).parent.absolute()
+    img_dir_path = str(cur_dir / "mock_data" / "coco" / "images")
+    h, v, d = get_wt_coeffs_var(img_dir_path)
+
+    assert h is not None
```

Comment:
wondering if h,v,d need to meet other requirements besides `not None`?

Reply:
updated test